pkg/release/release.go

```diff
@@ -19,11 +19,17 @@ package release
 var Base = "4.10.18"
 
 var Image = map[string]string{
-	"cli":                 "quay.io/microshift/cli:" + Base,
-	"coredns":             "quay.io/microshift/coredns:" + Base,
-	"haproxy_router":      "quay.io/microshift/haproxy-router:" + Base,
-	"kube_rbac_proxy":     "quay.io/microshift/kube-rbac-proxy:" + Base,
-	"pause":               "quay.io/microshift/pause:" + Base,
-	"service_ca_operator": "quay.io/microshift/service-ca-operator:" + Base,
-	"ovn_kubernetes":      "quay.io/microshift/ovn-kubernetes-singlenode:" + Base,
+	"cli":                     "quay.io/microshift/cli:" + Base,
+	"coredns":                 "quay.io/microshift/coredns:" + Base,
+	"haproxy_router":          "quay.io/microshift/haproxy-router:" + Base,
+	"kube_rbac_proxy":         "quay.io/microshift/kube-rbac-proxy:" + Base,
+	"pause":                   "quay.io/microshift/pause:" + Base,
+	"service_ca_operator":     "quay.io/microshift/service-ca-operator:" + Base,
+	"ovn_kubernetes":          "quay.io/microshift/ovn-kubernetes-singlenode:" + Base,
+	"odf_topolvm_rhel8":       "registry.redhat.io/odf4/odf-topolvm-rhel8@sha256:bd9fb330fc35f88fae65f1598b802923c8a9716eeec8432bdf05d16bd4eced64",
```

Comment:
This file is the fall-back for "community-supported architectures" where RH doesn't have official images.
It looks like the digests are the same as for the `amd64` architecture. Therefore, this isn't ever going to be correct.

I suggest either adopting the same approach as for the images above (`"quay.io/microshift/..." + Base`) even if those don't exist for now. Eventually, me may want to consider removing the `release.go` fallback altogether.

Reply:
Changed the value to reflect the public access image repos, which don't yet exist.